packages/code-infra/src/cli/cmdBuild.mjs

```diff
@@ -4,6 +4,7 @@ import set from 'lodash-es/set.js';
 import * as fs from 'node:fs/promises';
 import * as path from 'node:path';
 import { getOutExtension, isMjsBuild, validatePkgJson } from '../utils/build.mjs';
+import { copyHandler } from './cmdCopyFiles.mjs';
```

Comment:
Do we plan to keep the copy command around? Because if not I'd move all the logic to this file and just remove it.

Reply:
Keeping it for now to not make it a breaking change